.github/CODEOWNERS

```diff
@@ -109,6 +109,9 @@
 # PRLabel: %Purview
 /sdk/purview/ @qiaozha
 
+# PRLabel: %Personalizer
+/sdk/personalizer/ @sharathmalladi 
```

Comment:
@HarshaNalluru:  Is this a partner contact that needs to be included for issues when the `Service Attention` label is applied?

Reply:
Yes :)